src/database/invoices.rs

```diff
@@ -84,10 +87,16 @@ impl DatabaseConnection {
                     &invoice
                         .attachments
                         .into_iter()
-                        .map(|a| NewAttachment {
+                        .enumerate()
+                        .map(|(i, a)| NewAttachment {
                             invoice_id: created_invoice.id,
                             hash: a.hash,
                             filename: a.filename,
+                            description: invoice
+                                .attachment_descriptions
+                                .get(i)
+                                .unwrap_or(&String::new())
+                                .to_string(),
```

Comment:
Instead of `enumerate` and access via `.get`, just use `zip` instead? This way we get rid of the `unwrap`